src/coreclr/vm/CMakeLists.txt

```diff
@@ -855,7 +855,6 @@ elseif(CLR_CMAKE_TARGET_ARCH_ARM64)
     set(VM_SOURCES_DAC_AND_WKS_ARCH
         ${ARCH_SOURCES_DIR}/stubs.cpp
         exceptionhandling.cpp
-        gcinfodecoder.cpp
```

Comment:
Is this change intentionally part of this PR? It seems unrelated at a glance.

Reply:
Yes, there were some multiply defined symbols due to double inclusion of this file and LLD complained.